### ISSUE TITLE
[Feature Request]: Add identifier for when app is in fullscreen

### ISSUE THREAD

**Issue by kommunarr** [FreeTubeApp/FreeTube#5912]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

FreeTube gets a lot of issue reports by users who do not know that they have fullscreen mode on and/or do not know how to exit it. This is referring to the whole app — not a video — being open in fullscreen.

### Proposed Solution

Add some kind of identifier to the user experience that fullscreen is on and can be disabled. E.g., a toggle in Theme Settings to enable/disable fullscreen.

### Alternatives Considered

I'm not familiar with how other apps have handled this problem, but there could be other good solutions to this. If anyone does have a better idea in mind that another app employs, please share it.

### Issue Labels

ease of use improvement, improvement to existing feature

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
IMO i dont think that adding this is worth the hassle. The cause of this are https://github.com/FreeTubeApp/FreeTube/issues/4534 and https://github.com/FreeTubeApp/FreeTube/issues/3220. To users that are opening reports its kind of random yes but if we address these issues in https://github.com/FreeTubeApp/FreeTube/pull/5138 then we dont even need to communicate this.

**Comment by kommunarr** [comment]
I agree with those appearing to be two major causes, although people who have entered fullscreen through those bugs may not ever know it is a controllable/leaveable state even after we fix the original bugs, and people may occasionally press <kbd>F11</kbd> and land up in the same place of confusion. Maybe #2822 is a little closer to it if they can figure out that a keyboard shortcut could fix it.

**Comment by kommunarr** [comment]
You're very right, but to be fair to the users, it is inconvenient to expect people to read documentation for every problem, or even know how to find the documentation link easily. The ideal for apps is that you teach your users how to use them. With regards to this discussion, that might look like:
- Adding a Help section to the browser container that has a link to the documentation
- Adding a link to the documentation somewhere on the Settings page (where most people go to first for their regular "what's going on here" inquiries) too
- Having app fullscreen be a configurable Theme Setting instead of just a keyboard shortcut
